vendor/init.go

```diff
@@ -110,6 +170,7 @@ func main() {
 }
 
 func downloadFile(file file) (err error) {
+	var h hash.Hash
```

Comment:
Move this to just above the switch statement, since it only starts to get used there.